### PR TITLE
Use cookie value to decide whether to serve personalised ads

### PR DESCRIPTION
**Pull request by kelvin-chappell**
~~We will only serve personalised ads if the `noPersonalisedAds` switch is off and the cookie tells us to do it.~~
We will ***always*** serve personalised ads unless the cookie tells us ***not*** to do it.

~~I've called the switch `noPersonalisedAds` rather than `personalisedAds` because the latter suggests that switching it on would make all ads personalised.~~

~~This is provisional as cookie name will probably change.~~

### CODE REVIEW

**Comment by walaura** [comment]
we will indeed end up going with a smaller cookie (we can decide tomorrow) and I'll export it so you can import it to your code instead of typing it twice!

File: static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js [hunk 1]
```diff
@@ -66,9 +68,11 @@ const setDfpListeners = (): void => {
 };
 
 const setPersonalisedAds = (): void => {
+    const cookieName = 'GU_PERSONALISED_ADS_GOOGLE';
     if (config.switches.includePersonalisedAdsConsent) {
-        // TODO: replace this hardcoded value with one from local storage
-        const personalised = false;
```
Comment (walaura) [inline static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js]: With #19649 you'll be able to use a nicer `getAdConsentState(thirdPartyTrackingAdConsent)` that returns a normal boolean and abstracts away the cookie reading!

File: static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js [hunk 2]
```diff
@@ -1,40 +1,42 @@
...
 import { init as getPageTargeting } from 'commercial/modules/messenger/get-page-targeting';
+import { init as getStyles } from 'commercial/modules/messenger/get-stylesheet';
 import { init as hide } from 'commercial/modules/messenger/hide';
 import { init as resize } from 'commercial/modules/messenger/resize';
 import { init as scroll } from 'commercial/modules/messenger/scroll';
+import { init as type } from 'commercial/modules/messenger/type';
 import { init as viewport } from 'commercial/modules/messenger/viewport';
-import { init as sendClick } from 'commercial/modules/messenger/click';
-import { init as background } from 'commercial/modules/messenger/background';
-import { init as disableRefresh } from 'commercial/modules/messenger/disable-refresh';
```
Comment (walaura) [inline static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js]: what happened in this whole import list? 🙀

File: static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js (hunk 1 above)
Comment (kelvin-chappell) [reply static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js]: Sounds much better.

File: static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js (hunk 2 above)
Comment (kelvin-chappell) [reply static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js]: Just a bit of moving around

**Review by walaura — APPROVED** [review]

**Pull request merged**